nemo/collections/tts/models/fastpitch.py

```diff
@@ -195,13 +195,21 @@ def parser(self):
         return self._parser
 
     def parse(self, str_input: str, normalize=True) -> torch.tensor:
+        if self.training:
+            logging.warning("parse() is meant to be called in eval mode.")
```

Comment:
Great idea. Let's do the same in `generate_spectrogram`:

Remove it:
```
# FIXME: return masks as well?
 self.eval()
 ```
 
 And add it:
 
 ```
 if self.training:
            logging.warning("generate_spectrogram() is meant to be called in eval mode.")
 ````

Reply:
Done